feature-utils/poly-look/test/react-components/inputs/textField.test.js

```diff
@@ -0,0 +1,141 @@
+import React from "react";
+import { render, fireEvent, screen, waitFor } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import { TextField } from "../../../src/react-components/";
+
+describe("TextField", () => {
+  describe("rendering", () => {
+    it("basic", () => {
+      render(<TextField name="test" helperText="helper" label="label" />);
+
+      expect(document.querySelector("input")).toHaveAttribute("value", "");
+
+      expect(screen.getByText("helper")).toBeTruthy();
+      expect(screen.getByText("label")).toBeTruthy();
+      expect(screen.getByTestId("triangle-exclamation")).not.toBeVisible();
+    });
+
+    it("pre filled", () => {
+      render(<TextField value="test" />);
+      expect(document.querySelector("input")).toHaveAttribute("value", "test");
```

Comment:
Not totally sure this is needed. There does not seem to be any internal logic set in motion when you set a value.